### PR TITLE
add readme of doc

### PR DESCRIPTION
**Pull request by kevinintel**
## Type of Change

Refine readme and doc

## Description

Documentation

## Expected Behavior & Potential Risk

n/a

## How has this PR been tested?

n/a

## Dependency Change?

n/a

### CODE REVIEW

File: docs/smoothquant.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+```
+
+#### Per-channel limitation
+
+Though per-channel quantization could bring lower quantization error, we could not apply it for activations due to the difficulty of the dequantization. We would prove it in the following image and the zero point of quantization would be ignored for simplicity.
+
+The image on the left presents a normal linear forward  with 1x2 input $x$ and 2x2 weight $w$. The results $y$ could be easily obtained by simple mathematics. In the middle image, we apply per-tensor quantization for activations and per-channel quantization for weights; the results after quantization that are denoted by $y_1$ and $y_2$, could be easily dequantized to the float results $y_{fp1}$ and $y_{fp2}$ by per channel scale $1.0/s_1s_x$ and $1.0/s_2s_x$. However, after applying per-channel quantization for activation (right image), we could not dequantize the  $y_1$ and  $y_2$ to float results.
+
+<div align="center">
+    <img src="./imgs/sq_pc.png"/>
```
Comment (kevinintel) [inline docs/smoothquant.md:223]: bad link

File: docs/smoothquant.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+</div>
+
+
+
+For most of the models such as OPT and BLOOM, $\alpha = 0.5$ is a well-balanced value to split the difficulty of weight and activation quantization. A larger $\alpha$ value could be used on models with more significant activation outliers to migrate more quantization difficulty to weights.
+
+
+### Our enhancement: 
+
+#### Algorithm: Layer-wise Auto-tuning of $\alpha$.
```
Comment (kevinintel) [inline docs/smoothquant.md:274]: $\ ?

File: docs/smoothquant.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+<div align="center">
+    <img src="./imgs/sq_pc.png"/>
+</div>
+
+
+## SmoothQuant and Our Enhancement
+
+### SmoothQuant
+
+In the previous subsection, we have explained why per-channel quantization could not be applied for activation, even though it could lead to lower quantization loss. However, the quantization error loss of activation plays an important role in the accuracy loss of model quantization[^2][^3][^4].
```
Comment (kevinintel) [inline docs/smoothquant.md:231]: bad link

File: docs/smoothquant.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+## SmoothQuant and Our Enhancement
+
+### SmoothQuant
+
+In the previous subsection, we have explained why per-channel quantization could not be applied for activation, even though it could lead to lower quantization loss. However, the quantization error loss of activation plays an important role in the accuracy loss of model quantization[^2][^3][^4]. 
+
+
+
+To reduce the quantization loss of activations, lots of methods have been proposed. In the following, we briefly introduce SPIQ[^2], Outlier Suppression[^3] and Smoothquant[^4]. All these three methods share a similar idea to migrate the difficulty from activation quantization to weight quantization but differ in how much difficulty to be transferred.
```
Comment (kevinintel) [inline docs/smoothquant.md:235]: bad link

File: docs/smoothquant.md [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+
+
+So **the first question is how to migrate the difficulty from activation to weights?** The solution is straightforward, that is to convert the network to an output equivalent network that is presented in the image below and apply quantization to this equivalent network. The intuition is that each channel of activation could be scaled to make it more quantization-friendly, similar to a fake per-channel activation quantization.
+
+<div align="center">
+    <img src="./imgs/sq_convert.png"/>
+</div>
+
+
+Please note that this conversion will make the quantization of weights more difficult, because the scales attached to weights shown above are per-input-channel, while quantization of weights is per-output-channel or per-tensor.
```
Comment (kevinintel) [inline docs/smoothquant.md:245]: it's better to add the real case, ex: 
fig 4 in https://www.arxiv-vanity.com/papers/2211.10438/

**Comment by hshen14** [comment]
SmoothQuant md is mainly from INC's one. Shall we just keep a small paragraph to introduce SmoothQuant and link to INC one? Then we can copy the result table (from INC) to here. In the future, we can add more results in ITREX.

File: docs/smoothquant.md (hunk 2 above)
Comment (changwangss) [reply docs/smoothquant.md]: $\ is the latex format, $\alpha$  will show on README.

**Review by hshen14 — APPROVED** [review]

**Pull request merged**